fineract-core/src/main/java/org/apache/fineract/infrastructure/core/service/PagedRequest.java

```diff
@@ -44,7 +43,7 @@ public Optional<T> getRequest() {
         return Optional.ofNullable(request);
     }
 
-    public Pageable toPageable() {
+    public PageRequest toPageable() {
```

Comment:
Is there a reason to change this?

Reply:
Of course, to use the methods of PageRequest.